telemetry/csharp/AwsToolkit.Telemetry.Events/AwsToolkit.Telemetry.Events.csproj

```diff
@@ -3,11 +3,15 @@
 		<TargetFramework>net46</TargetFramework>
 		<RootNamespace>Amazon.AwsToolkit.Telemetry.Events</RootNamespace>
 		<AssemblyName>Amazon.AwsToolkit.Telemetry.Events</AssemblyName>
+		<PackageId>Amazon.AwsToolkit.Telemetry.Events</PackageId>
+		<Version>$(Version)</Version>
```

Comment:
What happens if a Version parameter is not provided? Should the csproj file have an element that defaults it to something like "0.0.1"?

Reply:
If I do not provide a parameter /p:Version when running the msbuild script, by default a AwsToolkit.Events.1.0.0.nupkg is created.

Reply:
Normally that would be great. Since this repo's [incremental version](https://github.com/aws/aws-toolkit-common/blob/master/version) is below 1.0.0 (currently 0.0.46), we'll probably want a default that remains pinned to a version less than the versions in use.

Something like this in the csproj file may pin the verison to 0.0.1 when the parameter is not available.

```
  <PropertyGroup>
    <Version Condition="'$(Version)' == ''">0.0.1</Version>
  </PropertyGroup>
```